api/LMPlatform.UI/Services/Practicals/PracticalService.svc.cs

```diff
@@ -354,24 +354,27 @@ public PracticalsResult GetPracticalsV2(int subjectId, int groupId)
                         Mark = string.Empty
                     })?.ToList()
                 }).ToList();
-                var durationCount = 0;
 
+                int durationCount = 0;
```

Comment:
Why convert a var into a specific type?

Reply:
This makes the code more understandable and accessible.

Reply:
Read basic practices and approaches.
The value determines the type. it makes it clear what type of data will be there.